RoadMap_SwiftUI/RoadMap_SwiftUI/ViewModel/NotificationViewModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  NotificationViewModel.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 27.01.2023.
+//
```

Comment:
```suggestion
//
//  NotificationViewModel.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 27.01.2023.
//
```
не отработал форматер